framework/src/outputs/TableOutput.C

```diff
@@ -50,7 +52,8 @@ TableOutput::TableOutput(const InputParameters & parameters) :
     _vector_postprocessor_time_tables(_tables_restartable ? declareRestartableData<std::map<std::string, FormattedTable> >("vector_postprocessor_time_table") : declareRecoverableData<std::map<std::string, FormattedTable> >("vector_postprocessor_time_table")),
     _scalar_table(_tables_restartable ? declareRestartableData<FormattedTable>("scalar_table") : declareRecoverableData<FormattedTable>("scalar_table")),
     _all_data_table(_tables_restartable ? declareRestartableData<FormattedTable>("all_data_table") : declareRecoverableData<FormattedTable>("all_data_table")),
-    _time_data(getParam<bool>("time_data"))
+    _time_data(getParam<bool>("time_data")),
+    _time_column(getParam<bool>("time_column"))
```

Comment:
@aeslaughter - you're pedantic about this kind of stuff, what's your take on this?